sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala

```diff
@@ -172,8 +182,11 @@ abstract class Output(keyName: String,
 
 object Output {
 
-  final val SEPARATOR = "_"
-  final val ID = "id"
+  final val Separator = "_"
+  final val Id = "id"
+  final val FixedAggregation = "fixedAggregation"
```

Comment:
My doubt is about FixedAggregation, are these aggregation fields global? or related to the output?